Prototyper/Classes/Feedback/EditScreenshotView.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  EditScreenshotView.swift
+//  Prototyper
+//
+//  Created by Raymond Pinto on 03.01.20.
+//
+
+import SwiftUI
+
+struct Drawing {
+    var points: [CGPoint] = []
+}
+
+struct EditScreenshotView: View {
+    @EnvironmentObject var model: Model
+    @Environment(\.presentationMode) var presentationMode
+    @Environment(\.colorScheme) var colorScheme
+    @State var currentDrawing: Drawing = Drawing()
+    @State var drawings: [Drawing] = [Drawing]()
+    @State var rect: CGRect = .zero
+    @State var color: Color = Color.black
+    @State var colorPickerShown: Bool = false
+    
+    var body: some View {
+        VStack(alignment: .center) {
+            GeometryReader { geometry in
+                Path { path in
+                    for drawing in self.drawings {
+                        self.add(drawing: drawing, toPath: &path)
+                    }
+                    self.add(drawing: self.currentDrawing, toPath: &path)
+                }
+                .stroke(self.color, lineWidth: 7)
+                .background(Image(uiImage: self.model.screenshot)
+                .resizable()
+                .scaledToFit())
+                .gesture(
+                    DragGesture(minimumDistance: 0.1)
+                        .onChanged({ (value) in
+                            let currentPoint = value.location
+                            if currentPoint.y >= 0
+                                && currentPoint.y < geometry.size.height {
+                                self.currentDrawing.points.append(currentPoint)
+                            }
+                        })
+                        .onEnded({ (value) in
+                            self.drawings.append(self.currentDrawing)
+                            self.currentDrawing = Drawing()
+                        })
+                )
+                
+            }.background(RectGetter(rect: $rect))
+                .navigationBarTitle("Markup View")
+                .navigationBarItems(leading: cancelButton, trailing: saveButton)
+            HStack (alignment: .center, spacing: 30) {
```

Comment:
I would suggest to add some spacer in before the HStack:
```suggestion
            Spacer(minLength: 16)
            HStack (alignment: .center, spacing: 30) {
```

Reply:
The Spacers are messing up the alignment of the image for some reason. But I've added padding instead and it's giving the View the desired look.